packages/mgt-components/src/components/mgt-profile/mgt-profile.ts

```diff
@@ -353,7 +355,7 @@ export class MgtProfile extends BasePersonCardSection {
            </div>
            <div class="data-list__item__content">
              <div class="educational-activity__degree" tabindex="0">
-               ${educationalActivity.program.displayName || 'Bachelors Degree'}
+               ${educationalActivity.program.displayName}
```

Comment:
I think that maybe we should avoid rendering this block if `educationalActivity.program.displayName` is falsy.

Perhaps something like:
```TypeScript
${
  educationalActivity.program.displayName
    ? html`<div class="data-list__item__content">
      <div class="educational-activity__degree" tabindex="0">
      ${educationalActivity.program.displayName}
    </div>`
    : nothing
}
```

Reply:
absolutely - will do (kept it because 'data-list__item__content's were not hidden in other places. first-pr-caution ;)